arangod/VocBase/LogicalCollection.cpp

```diff
@@ -404,7 +410,7 @@ ErrorCode LogicalCollection::getResponsibleShard(arangodb::velocypack::Slice sli
 }
 
 /// @briefs creates a new document key, the input slice is ignored here
-std::string LogicalCollection::createKey(VPackSlice) {
+std::string LogicalCollection::createKey(VPackSlice) const {
```

Comment:
It may work like this, but it feels wrong that `createKey()` is const.
I guess the keyGenerator will need to modify its internal state somehow, so it feels non-consty.

Reply:
I agree, no idea why it compiled this way.
Removed the const